src/bp-templates/bp-nouveau/includes/messages/ajax.php

```diff
@@ -420,7 +420,7 @@ function bp_nouveau_ajax_get_user_message_threads() {
 	// Remove the bp_current_action() override.
```

Comment:
According to ticket's comment (`#30`) I believe you might need to add something like this:

[9160.patch](https://github.com/user-attachments/files/17347335/9160.patch)